exonum-java-binding-core/src/main/java/com/exonum/binding/util/OsInfo.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.util;
+
+class OsInfo {
+
+  private static final boolean onWindows;
+
+  static {
+    String osName = System.getProperty("os.name").toLowerCase();
+    onWindows = osName.contains("win");
```

Comment:
Why not `windows`? I'd expect that to be good enough for any Windows OS. Is `win` always correct, won't there be other than Windows OSes that match?

Reply:
I think we can include "windows" for clarity.